stock_custom_exchange_rate_date/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Customized Rate Date in Stock",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo",
+    "category": "stock",
+    "website": "http://www.vauxoo.com/",
+    "license": "LGPL-3",
+    "depends": [
+        "stock",
```

Comment:
@luisg123v 

what about only purchase_stock module?

Reply:
I was trying to avoid depending on that module because it's an autoinstall one, but I can change it if you want.

Reply:
the correct dependency is purchase_stock (it is the one with the behavior we are overwritting here)

Reply:
Done